src/pages/Home.vue

```diff
@@ -84,8 +105,13 @@ export default {
     },
   },
   async created() {
-    /** Group name is sent to the API to retrieve all details */
-    this.groupData = await groupAPIService.getGroupData(this.group);
+    /** If sessionId exists in route, then retrieve session details. Otherwise, fallback is using group. */
+    if(this.sessionId === ""){
+      this.groupData = await groupAPIService.getGroupData(this.group); }
+    else{
+     this.sessionData = await sessionAPIService.getSessionData(this.sessionId);
+      (this.sessionData.enabled == "on") ? this.sessionEnabled = true : this.sessionEnabled = false;
```

Comment:
You can re-write it like this
```suggestion
      this.sessionEnabled = (this.sessionData.enabled == "on") ? true : false;
```